nautobot/core/templates/generic/object_import.html

```diff
@@ -15,19 +15,24 @@ <h1>{% block title %}{{ obj_type|bettertitle }} Import{% endblock %}</h1>
                 </div>
             </div>
         {% endif %}
-		<form action="" method="post" class="form form-horizontal">
-		    {% csrf_token %}
-		    {% render_form form %}
-            <div class="form-group">
-                <div class="col-md-12 text-right">
-                    <button type="submit" name="_create" class="btn btn-primary">Submit</button>
-                    <button type="submit" name="_addanother" class="btn btn-primary">Submit and Import Another</button>
-		            {% if return_url %}
-                        <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
-                    {% endif %}
-                </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Device Type</strong></div>
+            <div class="panel-body">
+		        <form action="" method="post" class="form form-horizontal object-import-form">
+                    {% csrf_token %}
+                    {% render_form form %}
+                    <div class="form-group">
+                        <div class="col-md-12 text-right">
+                            <button type="submit" name="_create" class="btn btn-primary">Submit</button>
+                            <button type="submit" name="_addanother" class="btn btn-primary">Submit and Import Another</button>
+                            {% if return_url %}
+                                <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
+                            {% endif %}
+                        </div>
+                    </div>
+                </form>
```

Comment:
Please remember to provide updated screenshots in the PR when making UI changes.

I pulled this down locally to try it out and it doesn't look quite right to me - I think a more correct nesting of elements, more consistent with `object_edit.html`, would be something like:

```html
<form>
    {% csrf_token %}
    <div class="panel panel-default">
        <div class="panel-heading">{{ obj_type|bettertitle }}</div>
        <div class="panel-body">
            {% render_form form %}
        </div>
    </div>
    <div class="form-group text-right">
        <!-- buttons... -->
    </div>
</form>
```